src/features/ProtectTransaction/ProtectTxProvider.tsx

```diff
@@ -8,10 +8,11 @@ import {
   GetTokenTxResponse
 } from '@services/ApiService';
 import { AssetContext, getAssetByUUID, StoreContext } from '@services/Store';
+import { NansenService, NansenServiceEntry } from '@services/ApiService/Nansen';
 import { useScreenSize } from '@utils';
+import { WALLETS_CONFIG } from '@config';
 
-import { WALLETS_CONFIG } from '../../config';
-import { NansenService, NansenServiceEntry } from '@services/ApiService/Nansen';
+import { IS_ACTIVE_FEATURE } from '../../config/isActiveFeature';
```

Comment:
Import from @config please

Reply:
It's imported directly to prevent circular dependency issues.